### PR TITLE
Embraser01/rewrite try fixes

### PR DESCRIPTION
**Pull request by mavimo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,8 +13,10 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./bin/ingress-controller .
 FROM alpine:latest AS certs
 RUN apk --update add ca-certificates
 
-FROM scratch
+FROM alpine:3.13.1
 COPY --from=builder /app/bin/ingress-controller .
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+RUN mkdir -p /etc/caddy/certs
+RUN mkdir -p ~/.config/caddy/
```
Comment (Embraser01) [inline Dockerfile:20]: We don't use an empty image anymore?

**Review by Embraser01 — COMMENTED** [review]
These changes are breaking compatibility with K8S < v1.19 clusters because of Ingress v1beta -> v1 updgrade.
The new version will also need a related IngressClass in the helm chart I think

File: Dockerfile (hunk 1 above)
Comment (mavimo) [reply Dockerfile:20]: @Embraser01 I'm try to understand if there is some impact, there are a few error on caddy related to the missing nss (but seems not required), having a sh support in docker image help me to debug the running container :P

**Comment by mavimo** [comment]
@Embraser01 AFAIK v1 is supported also in v1.19 and they already deprecated the v1beta1

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#ingress-v1-networking-k8s-io

**Review by Embraser01 — APPROVED** [review]

**Pull request closed without merging**